docs/reference/remote-build.rst

```diff
@@ -0,0 +1,56 @@
+.. _ref-remote-build:
+
+
+*************
+Remote builds
+*************
+
+Remote build is a feature in Rockcraft that offloads the build process to
+`Launchpad`_'s `build farm`_ to build multiple rocks simultaneously, and for
+different architectures than your running machine.
+
+
+Pre-requisites
+--------------
+
+In order to perform remote builds, the following conditions must be met:
+
+- You must have a `Launchpad account`_, as the remote builds are performed on
+  Launchpad.
+- The Rockcraft project must be version-controlled by Git. This is because
+  Rockcraft uses a Git-based workflow to upload the project to Launchpad.
+- The repository hosting the Rockcraft project must not be a shallow clone,
+  because Git does not support pushing shallow clones.
+
+
+Overview
```

Comment:
How about "How it works" or simply "Remote build process"?

I don't think the content quite fits the definition of overview, and typically an overview would come before prerequisites.

Reply:
I see your point. I think a section like "How it works" and "Remote build process" feels a bit... generic and redundant with the page itself, so I removed this section and put this paragraph in the "introduction", before the Prerequisites section. Looks like it might flow better like this.

Let me know what you think!

Reply:
Looks good. :)